include/grpcpp/security/tls_credentials_options.h

```diff
@@ -146,6 +146,41 @@ class TlsServerAuthorizationCheckConfig {
       server_authorization_check_interface_;
 };
 
+/// Configuration for Tls key logging.
+struct TlsKeyLoggerConfig {
+
+  // The full path at which the TLS keys would be exported.
+  std::string tls_key_log_file_path;
```

Comment:
We need to be C89 for all public header files.

Reply:
Resolving this since this is in the include/grpcpp directory containing c++ files